### ISSUE TITLE
Architectural distortion of breast

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10085]
**Preferred term label:**
Architectural distortion of breast

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C1268716

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mammary Paget disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000057
- label: Architectural distortion of breast
- parents: [Abnormal breast morphology (HP:0031093)](https://hpo.jax.org/app/browse/term/HP:0031093)
- pmid: PMID:32953609
- def: Architectural distortion is defined as an architectural disruption of the normal breast that does not have obvious mass opacities, which may be early signs of breast carcinoma.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10085
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
HPO ID corrected to HP:0040404
